site/app/models/gradeable/GradeableList.php

```diff
@@ -312,10 +312,10 @@ public static function getGradeableSection(Core $core, Gradeable $gradeable): in
         ) {
             return self::OPEN;
         }
-        elseif ($core->getUser()->accessGrading() && $gradeable->getTaViewStartDate() <= $now) {
+        elseif ($gradeable->getTaViewStartDate() <= $now) {
             return self::BETA;
         }
-        elseif ($core->getUser()->accessAdmin()) {
+        elseif ($gradeable->getTaViewStartDate() > $now) {
```

Comment:
Would changing these to not include if the user can access them make everyone be able to access them?

Reply:
So, let me explain this clearly
There are only two instances of GradeableList which are being used : 

1. `Submitty\site\app\controllers\NavigationController.php`, as you can see in the below image
Access is getting checked already for both beta and future gradeables and is getting displayed on gradeables page as per the logged in user level

![image](https://github.com/Submitty/Submitty/assets/81346327/78a4a47b-17af-4104-b529-28c23c745c43)


2. `Submitty\site\app\models\CalendarInfo.php`, as you can see in the below image

![image](https://github.com/Submitty/Submitty/assets/81346327/3e1d1035-1ea8-47c7-abc3-46c3097e7b46)

Access check is not necessary for this, because calendar is only used to display future/beta gradeables  for the students but not to access them with the help of below implementation in `calendar.js`
![image](https://github.com/Submitty/Submitty/assets/81346327/6f4d5b58-e70d-46f9-b14b-3039e5fec49a)

Reply:
Let me know if its not clear @IDzyre